config/resource-manager.hcl

```
@@ -468,6 +468,10 @@ service "solutions" {
   name      = "ManagedApplications"
   available = ["2019-07-01", "2021-07-01"]
 }
+service "sql" {
+  name      = "MsSql"
```

can we keep this as `Sql` rather than `MsSql` here, because of `SqlVirtualMachine` below?

```suggestion
  name      = "Sql"
```